rtwcli/README.md

```diff
@@ -7,3 +7,15 @@
 `source ros_team_workspace/setup.bash`
 
 `setup-auto-sourcing`
+
+## porting workspace
+
+add explicit export to variables: `bash ros_team_workspace/scripts/update-rtw.bash`
+
+source new methods with exports: `source ~/.ros_team_ws_rc`
+
+source your workspace: `_<ws_name>`
+
+port config: `rtw wokspace port`
```

Comment:
When I try to execute this I get the following error:

```
Failed to load entry point 'port': No module named 'questionary'
Traceback (most recent call last):
File "/usr/local/bin/rtw", line 33, in <module>
sys.exit(load_entry_point('rtwcli', 'console_scripts', 'rtw')())
File "/opt/RosTeamWS/ros_team_workspace/rtwcli/rtwcli/rtwcli/cli.py", line38, in main
add_subparsers_on_demand(
File "/opt/RosTeamWS/ros_team_workspace/rtwcli/rtwcli/rtwcli/command/__init__.py", line 189, in add_subparsers_on_demand
extension.add_arguments(command_parser, f"{cli_name} {name}", **kwargs)
File "/opt/RosTeamWS/ros_team_workspace/rtwcli/rtw_cmds/rtw_cmds/commands.py", line 27, in add_arguments
add_subparsers_on_demand(parser, cli_name, "_verb", self.verbs_group, required=False)
File "/opt/RosTeamWS/ros_team_workspace/rtwcli/rtwcli/rtwcli/command/__init__.py", line 176, in add_subparsers_on_demand
extension = command_extensions[name]
KeyError: 'port'
```

Do you have any idea why? (The same for `port_all`)